syscall.go

```diff
@@ -111,6 +114,62 @@ func unmountShareDir() error {
 	return os.RemoveAll(containerMountDest)
 }
 
+func waitForBlockDevice(deviceName string) (err error) {
+	devicePath := filepath.Join(devPath, deviceName)
+
+	if _, err := os.Stat(devicePath); err == nil {
```

Comment:
I guess that is true. If we do go that route, obviously put a small nap/sleep in the loop so we don't hard spin on it.